Source/WebGPU/WebGPU/HardwareCapabilities.mm

```diff
@@ -49,7 +49,10 @@ static constexpr auto multipleOf4(auto input)
 }
 static uint64_t maxBufferSize(id<MTLDevice> device)
 {
-    auto result = std::max<uint64_t>(defaultMaxBufferSize, std::min<uint64_t>(INT_MAX, device.maxBufferLength / 10));
+    constexpr uint64_t oneGigabyte = 1024 * 1024 * 1024;
```

Comment:
just fyi, but wtf/StdLibExtras.h defines `GB`.

Reply:
I will switch to GB but I wonder if I should even land this change. On iOS, the web content process will quickly get jetsam'ed when we report that a 1GB buffer can be created.